emeis/core/views.py

```diff
@@ -14,6 +16,23 @@ def get_object(self, *args, **kwargs):
         return self.request.user
 
 
+class MyACLViewSet(RetrieveModelMixin, ListModelMixin, GenericViewSet):
+    """MyACL view returns current users ACLs."""
+
+    queryset = models.ACL.objects.all()
+    serializer_class = serializers.MyACLSerializer
+
+    def get_object(self):
+        pk = self.kwargs.pop("pk")
+        acl = get_object_or_404(models.ACL, pk=pk)
+        if acl.user != self.request.user:
+            raise PermissionDenied()
```

Comment:
Would it make sense to handle this in permissions? The name of the exception is kinda leading there...

Reply:
Not in this view: Here it's the express purpose that you're only ever see your own ACLs.